src/url.js

```diff
@@ -64,6 +65,31 @@ export async function redirectIfLoggedIn({ redirect } = {}) {
   }
 }
 
+/**
+ * If the access token is invalid, redirect the browser to the
+ * provided path.
+ */
+export async function redirectIfLoggedOut({ redirect } = {}) {
+  // If the user is logged in, return without doing anything
+  const isUserLoggedIn = await isLoggedIn();
+  if (isUserLoggedIn) {
+    return;
+  }
+
+  // Remove all cookies
+  removeAllCookies();
```

Comment:
Should the function name & docstring mention this? I think it's fine to remove cookies here, but a client could rely on cookies in a way that we don't expect. Worst case for the client would be a race condition causing flaky behavior if they do `Promise.all([doSomethingWithCookies, redirectIfLoggedOut])`

Like above though, having trouble finding the right balance between simplicity and description in naming. `cleanupAndRedirectIfLoggedOut` is a bit wordy... it's probably OK named as is, if the docstring & documentation calls out that this clears cookies too.

Reply:
Despite the name, this only removes cookies related to the session: access, refresh, and id. If we have determined that the user is no longer logged in, then those cookies should not be considered valid.

- [ ] Will make sure to note that it cleans up the cookies in this case in the documentation.